sources/web/datalab/polymer/test/item-list-test.ts

```diff
@@ -500,6 +515,145 @@ describe('<item-list>', () => {
       assert(testFixture.$.listContainer.querySelectorAll('.row').length === 5,
           'all rows should show after closing filter box');
     });
+  });
+
+  describe('sorting', () => {
+    const rows = [
+      new ItemListRow({columns: ['item c*', 'Sat Nov 11 2017 18:58:42 GMT+0200 (EET)']}),
```

Comment:
Looks like these date strings should now be Date values instead, and change the type on line 539 to DATE.

Reply:
Done.